constants/crawl.go

```diff
@@ -0,0 +1,5 @@
+package constants
+
+const (
```

Comment:
I already have a `constants` package. I am keeping it here as I am using it multiple times in test